src/application.ts

```diff
@@ -77,7 +78,8 @@ export class PocketGatewayApplication extends BootMixin(ServiceMixin(RepositoryM
     const defaultLogLimitBlocks: number = parseInt(DEFAULT_LOG_LIMIT_BLOCKS) || 10000
     const aatPlan = AAT_PLAN || AatPlans.PREMIUM
     const redirects: string | object[] = REDIRECTS || ''
-    const commitHash: string | string = COMMIT_HASH || ''
+    const commitHash: string = COMMIT_HASH || ''
+    const archivalChains: string[] = (ARCHIVAL_CHAINS || []).split(',')
```

Comment:
Will this fail if someone accidentally puts a space after the commas?

Reply:
not fail per se but will add the space to the result, will add a small fix for that